ocaml/xapi/nm.ml

```diff
@@ -566,6 +566,7 @@ let bring_pif_up ~__context ?(management_interface = false) (pif : API.ref_PIF)
               Option.iter
                 (fun (_, name) -> Net.set_dns_interface dbg name)
                 dns_if ;
+              Net.write_config () ;
```

Comment:
This can't be done until all the `Net.*.make_config` calls have completed, at around line 693